utils/scripts/install-centos7.sh

```diff
@@ -57,3 +57,4 @@ dnf --nodocs install \
     sudo \
     valgrind-devel \
     yasm
+ln -s /usr/bin/cmake3 /usr/bin/cmake
```

Comment:
Simplest workaround for a platform we are discontinuing.

Reply:
I don't think that will do what you want.  That is altering the environment that `mock` (the tool that builds the chroot that the RPM build is done in) is running in, not the environment that `rpmbuild` runs in.  There is no way to achieve your simple hack above in that chroot.

Reply:
More to the point, how did you even figure out you needed this if we are not building on CentOS 7 any more?

Reply:
Because we still run unit tests on Centos 7